wvd-templates/Update existing WVD host pool/mainTemplate.json

```diff
@@ -3,7 +3,7 @@
     "contentVersion": "1.0.0.0",
     "parameters": {
         "_artifactsLocation": {
-            "defaultValue": "https://raw.githubusercontent.com/Azure/RDS-Templates/master/wvd-templates/Update%20existing%20WVD%20host%20pool",
+            "defaultValue": "https://raw.githubusercontent.com/Azure/RDS-Templates/master/wvd-templates/",
```

Comment:
Can you end the "defaultValue" without the backslash? I remember that marketplace broke when backslash was there.

Reply:
Ignore. Suggestions are already aligned with "Create".